### PR TITLE
urdf clean up

### PR DESCRIPTION
**Pull request by khizarmohammed1997**

### CODE REVIEW

File: alex_description/urdf/20240808_Alex_WithSpine_Nubs.urdf
Comment (christiandebuys) [inline alex_description/urdf/20240808_Alex_WithSpine_Nubs.urdf:1]: I would remove "WithSpine". The default should be that the robot has all its joints, and we say what it's missing in the urdf and enum names.

File: alex_description/urdf/20240808_Alex_Spineless_BarrettHands.urdf
Comment (christiandebuys) [inline alex_description/urdf/20240808_Alex_Spineless_BarrettHands.urdf:1]: Really just an opinion, but "Spineless" sounds silly to me. Maybe "NoSpine", "NoHands", etc. Just "No[ThingThatIsMissing]"

File: alex_description/meshes_backup/Torso_Updated.obj
Comment (christiandebuys) [inline alex_description/meshes_backup/Torso_Updated.obj:1]: What is "updated" here? If these are the new files, I would rename Torso.mtl and Torso.obj to "_old" and maybe move them somewhere else. Then you can call the updated ones just "Torso".

File: alex_description/urdf/20240808_Alex_Spineless_BarrettHands.urdf
Comment (christiandebuys) [reply alex_description/urdf/20240808_Alex_Spineless_BarrettHands.urdf:1]: The Barrett file has the dumbbells in the urdf, but that is not reflected in the enum name. Should have "WithDumbbells" and could probably just make it a separate urdf.

File: alex_description/urdf/20240808_Alex_WithSpine_Nubs.urdf
Comment (christiandebuys) [reply alex_description/urdf/20240808_Alex_WithSpine_Nubs.urdf:1]: Also this guy seems to have some issues:
image

**Review by christiandebuys — CHANGES_REQUESTED** [review]

File: alex_description/urdf/20240808_Alex_WithSpine_Nubs.urdf
Comment (khizarmohammed1997) [reply alex_description/urdf/20240808_Alex_WithSpine_Nubs.urdf:1]: As discussed, with spine URDF will not be accurate at all

File: alex_description/urdf/20240808_Alex_Spineless_BarrettHands.urdf
Comment (khizarmohammed1997) [reply alex_description/urdf/20240808_Alex_Spineless_BarrettHands.urdf:1]: Fixed these

File: alex_description/meshes_backup/Torso_Updated.obj
Comment (khizarmohammed1997) [reply alex_description/meshes_backup/Torso_Updated.obj:1]: Removed them

**Review by christiandebuys — APPROVED** [review]

**Pull request merged**